modules/devices.nix

```diff
@@ -35,9 +35,11 @@ let
     (lib.concatLines (map
       # all files in linux-firmware are read-only
       (firmwarePath: ''
-        install -Dm0444 \
-          --target-directory=$(dirname $out/lib/firmware/${firmwarePath}) \
-          $(realpath ${pkgs.linux-firmware}/lib/firmware/${firmwarePath})
+        if [ -f ${pkgs.linux-firmware}/lib/firmware/${firmwarePath} ]; then
```

Comment:
Some of the files in linux-firmware are symlinks, I don't think `-f`still works in that case. Can we also pull this change into a separate PR?

Reply:
Yea, the change is just so I can test this PR. I intend to remove it.